tests/fixtures/mock_curl.php

```diff
@@ -27,15 +27,40 @@
 
 class mock_curl extends \curl {
 
-    public function __construct($responsecode, $response) {
+    /**
+     * Is this curl for testing retries?
+     *
+     * @var bool
+     */
+    private $retrytest;
+
+    /**
+     * Counter for sequential calls.
+     *
+     * @var int
+     */
+    public $callcount;
```

Comment:
As the descriptions are very short and simple, could we change the format of this to also be single line comments?
```suggestion
    /** @var bool Is this curl for testing retries? */
    private $retrytest;

    /** @var int Counter for sequential calls */
    public $callcount;
```